lib/jekyll/entry_filter.rb

```diff
@@ -33,7 +36,9 @@ def filter(entries)
     end
 
     def included?(entry)
-      glob_include?(site.include, entry)
+      glob_include?(site.include,
+        entry
+      )
```

Comment:
Which is it – both arguments on a new line or just the second? Let's get a rubocop rule in for this.